docs/reference/ccr/apis/auto-follow/put-auto-follow-pattern.asciidoc

```diff
@@ -58,7 +58,10 @@ This API creates a new named collection of
 <<ccr-auto-follow,auto-follow patterns>> against the remote cluster
 specified in the request body. Newly created indices on the remote cluster
 matching any of the specified patterns will be automatically configured as follower
-indices. Additionally, this API can be used to update existing
+indices. Indices on the remote cluster that have been created before the auto-follow
+ pattern is created won't be auto-followed even if they match the pattern.
```

Comment:
Small fix:

```suggestion
indices. Indices on the remote cluster that were created before the auto-follow
pattern is created won't be auto-followed even if they match the pattern.
```

or perhaps this?

```suggestion
indices. Indices on the remote cluster that exist when the auto-follow
pattern is created won't be auto-followed even if they match the pattern.
```

Reply:
I borrowed the first suggestion, thanks David!